content/community-targets/NESHTEC-NeshNode.md

```diff
@@ -0,0 +1,96 @@
+# NESHTEC NeshNode
+
+NeshNode is a flexible, high performance board featuring a powerful STM32F7 dual core RISC based microcontroller, on-board ethernet, battery charger, lots of I/Os, two MikroE® Click™ sockets and a lot of tiny details, like a robust USB-C connector.
+
+Furthermore, our board has been extensively tested by the independent testing institute [TÜV Nord](https://www.tuev-nord.de/) for EMC/EMI (electromagnetic compatibility / interference), which means that it is fully compliant with EU regulations (CE).
+
+![NESHTEC NeshNode](../../images/community-targets/NESHTEC-NeshNode-STM32F7-1.1-nf.png)
+
+## How to get one?
+Just check out our shop, we'll ship the board from Germany:
+https://neshtec.eu/products/neshnode-controllerplatine
+
+(We are working on a English translation, sorry about that)
+
+## Hardware
+
+NeshNode was designed to be used as flexible as possible in professional projects, without having to develop a new board every time. The board is powerful enough for extensive cryptographic tasks, cloud connectivity (like Microsoft Azure IoT Hub) and offers a lot of interfaces.
+
+### STM32F765VGT6 microcontroller
+
+The board currently uses an ARM Cortex-M7 RISC core based STM32F765VGT6 MCU in a 100 pin package which operates at up to 216 MHz, 2 MB of persistent flash, 512KB SRAM, features a dedicated double precision FPU and also implements a full set of DSP instructions. To increase application security, it also features a MPU (memory protection unit).
+
+See here for more details: https://www.st.com/en/microcontrollers-microprocessors/stm32f765vg.html
+
+### Ethernet
+
+Ethernet is supplied by a Microchip LAN8720A, which is a high performance 10BASE-T/100BASE-TX transceiver delivering up to 100 MBit/s, working on a large temperature range: https://www.microchip.com/en-us/product/lan8720a
+
+### MikroE® Click™ / MikroBus™
+
+MikroE® has designed a compact interface for which, as of today, there are over 1,000 different so-called Click™ boards of very high quality - giving you the flexibility to easily add whatever kind of additional functionality you might need.
+
+See below ([Pinout](#pinout)) for a sample of MikroE®s MikroBus™ layout.
+
+Check out MikroE®'s Click™ board store for some awesome addon boards: https://www.mikroe.com/click
+
+More information about the Click™ standard can be found on the MikroE®s website: https://www.mikroe.com
+
+### USB-C connector
```

Comment:
Markdown 1à1: you need a new line before and after each title (except the main one if at the fist line of the doc)
```suggestion
### USB-C connector

```

Reply:
Yeah, already checked your pipelines linter log. I'm about to fix that